### ISSUE TITLE
FailedSignInTest() passes even when it signs in

### ISSUE THREAD

**Issue by tscheffel** [microsoft/SmartHotel360-Mobile#78]
The Tests.cs file contains the following test which passes as expected when logging in fails:

```
        [Test]
        public void FailedSignInTest()
        {
            new LogInPage()
                .EnterCredentials(string.Empty, string.Empty)
                .SignIn();
        }
```

However, that test also passes when logging in succeeds using the following code.  I expected it to fail.

```
        [Test]
        public void FailedSignInTest()
        {
            new LogInPage()
                .EnterCredentials(TestSettings.TestUsername, TestSettings.TestPassword)
                .SignIn();
        }
```
I haven't yet figured out how the test is supposed to work without explicit Asserts (which is what I'm used to when working with NUnit). I see Asserts buried deeper in the code but it's not obvious when looking at the actual test what it expects to happen to make it fail (I expected to see an Assert confirming the presence of a login error message).

Bottom line: why does the test pass even when login succeeds?

**Comment by pulimento** [comment]
You're right. We need to look at it. Thanks for pointing it out :)

**Comment by tscheffel** [comment]
Roger that, thanks for looking into it.  I'd fix the code issue myself if I understood how the test is supposed to work without Assert statements.  I have yet to find a resource on the way you guys are writing your tests (fluent-style with no obvious Asserts; perhaps they are buried deep in the hierarchy).

**Comment by MarcosCobena** [comment]
Hey @tscheffel,

You were completely right, I've tried to improve it to actually check there's been no navigation in such case.

Thanks for pointing that out,

—Marcos
